### PR TITLE
ALEX, Kyber

### PR DESCRIPTION
**Pull request by waynebruce0x**

### CODE REVIEW

**Comment by llamatester** [comment]
The adapter at projects/alexlab exports TVL: 
        
 
 
    stacks                    13.66 M
    
    total                    13.66 M

File: projects/alexlab/index.js [hunk 1]
```diff
@@ -1,44 +1,21 @@
...
-const STACKS_API = 'https://stacks-node-api.mainnet.stacks.co/extended/v1/address'
+const ALEX_API = "https://api.alexlab.co/v1";
 
-const transformAddress = addr => {
-  const slug = addr.substr(addr.indexOf('::') + 2);
-  return slug
+async function fetch() {
+  const url = `${ALEX_API}/stats/tvl`;
+  const alexResponse = (await retry(async () => await axios.get(url))).data;
+  return alexResponse.lp_token_supply;
```
Comment (hyossid) [inline projects/alexlab/index.js:11]: should change to alexResponse.tvl

**Comment by hyossid** [comment]
do you mind changing this?

**Comment by llamatester** [comment]
The adapter at projects/alexlab exports TVL: 
        
 
 
    stacks                    12.55 M
    
    total                    12.55 M

File: projects/alexlab/index.js (hunk 1 above)
Comment (waynebruce0x) [reply projects/alexlab/index.js:11]: wouldnt that be double counting the LPs though (once in the AMM, and again when staked)

**Comment by llamatester** [comment]
The adapter at projects/alexlab exports TVL: 
        
 
 
    stacks                    12.55 M
    
    total                    12.55 M

**Comment by llamatester** [comment]
Error while running adapter at projects/kyber: 
        
 
  
    
    
    deployment `kybernetwork/kyberswap-elastic-cronos` does not exist
    Truncated error stack:
        at /home/runner/work/DefiLlama-Adapters/DefiLlama-Adapters/node_modules/graphql-request/dist/index.js:340:31

**Comment by llamatester** [comment]
The adapter at projects/kyber exports TVL: 
        
 
 
    arbitrum                  301.12 k
    oasis                     269.36 k
    aurora                    0.0000010210527441365309
    avax                      15.62 M
    optimism                  54.192672234910745
    ethereum                  26.45 M
    bsc                       2.97 M
    fantom                    35.39 k
    cronos                    37.32 k
    polygon                   10.26 M
    
    total                    55.94 M

File: projects/alexlab/index.js (hunk 1 above)
Comment (waynebruce0x) [reply projects/alexlab/index.js:11]: at defillama we only count the inital deposit to the LP pool and dont count further staking of the LPs. We use this method for all protocols

**Comment by llamatester** [comment]
The adapter at projects/alexlab exports TVL: 
        
 
 
    stacks                    12.98 M
    
    total                    12.98 M

**Comment by llamatester** [comment]
The adapter at projects/kyber exports TVL: 
        
 
 
    avax                      15.62 M
    oasis                     269.36 k
    optimism                  54.19267223491074
    arbitrum                  301.12 k
    bsc                       2.97 M
    ethereum                  26.45 M
    aurora                    0.0000010210527441365309
    cronos                    37.32 k
    fantom                    35.39 k
    polygon                   10.26 M
    
    total                    55.94 M

**Comment by llamatester** [comment]
The adapter at projects/alexlab exports TVL: 
        
 
 
    stacks                    12.98 M
    
    total                    12.98 M

**Comment by llamatester** [comment]
The adapter at projects/increment-lending exports TVL: 
        
 
 
    flow                      287.43 k
    
    total                    287.43 k

**Comment by llamatester** [comment]
The adapter at projects/kyber exports TVL: 
        
 
 
    avax                      15.62 M
    oasis                     269.36 k
    bsc                       2.97 M
    arbitrum                  301.12 k
    aurora                    0.0000010210527441365309
    optimism                  54.192672234910745
    ethereum                  26.45 M
    cronos                    37.32 k
    fantom                    35.39 k
    polygon                   10.26 M
    
    total                    55.94 M

**Pull request merged**